platforms/opengl/Makefile

```diff
@@ -1,8 +1,8 @@
 CC=gcc
-SOURCES=video.c input.c ../../common/main.c ../../common/minefield.c ../../common/video-tiles.c
+SOURCES=extras.c video.c input.c ../../common/main.c ../../common/minefield.c ../../common/video-tiles.c
 BUILDDIR=build
-CFLAGS=-Wall -pedantic -I../../common -I. -DNO_UNUSED_MACRO $(shell pkg-config --cflags gl sdl)
-LDFLAGS = $(shell pkg-config glu sdl --libs) -lglut -lm
+CFLAGS=-Wall -pedantic -I../../common -I. -DNO_UNUSED_MACRO -DMINEFIELD_ALLOCATION $(shell pkg-config --cflags gl sdl)
+LDFLAGS = $(shell pkg-config glu sdl --libs) -lGL -lGLU -lglut -lm
```

Comment:
`pkg-config glu --libs` is supposed to result in `-lGL -lGLU`:
![Screenshot from 2022-06-20 18-25-12](https://user-images.githubusercontent.com/213676/174679586-049de369-e1bc-4f80-b244-d699126f4d5f.png)

Reply:
I found this problem when using Debian: https://stackoverflow.com/questions/33017325/why-freeglut3-dev-doesnt-provide-pc-file-for-pkg-config

Reply:
Interestingly, we don't actually need **sdl**. It's there because I thought I would use it, but I ended up forgetting to remove it from the Makefile.